packages/web-react/src/components/FileUploader/useFileUploaderInput.ts

```diff
@@ -0,0 +1,131 @@
+import { ChangeEvent, DragEvent, useState } from 'react';
+import { useFileUploaderContext } from './FileUploaderContext';
+import {
+  DEFAULT_ERROR_MESSAGE_MAX_FILE_SIZE,
+  DEFAULT_ERROR_MESSAGE_QUEUE_DUPLICITY,
+  DEFAULT_ERROR_MESSAGE_QUEUE_LIMIT,
+} from './constants';
+
+export interface UseFileUploaderInputProps {
+  accept?: string;
+  fileLimitMaxSize: number;
+  fileQueueLimit: number;
+  isMultiple?: boolean;
+  onError?: (error: string) => void;
+}
+
+export interface UseFileUploaderInputReturn {
+  isDragging: boolean;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  onDragEnter: () => void;
+  onDragLeave: () => void;
+  onDragOver: (event: DragEvent<HTMLDivElement>) => void;
+  onDrop: (event: DragEvent<HTMLDivElement>) => void;
+}
+
+export const useFileUploaderInput = (props: UseFileUploaderInputProps): UseFileUploaderInputReturn => {
+  const { fileLimitMaxSize, fileQueueLimit, isMultiple, onError } = props;
+
+  const [isDragging, setDragging] = useState(false);
+
+  const { fileQueue, addToQueue, clearQueue } = useFileUploaderContext();
+
+  const getUpdatedFileName = (name: string): string => `attachment__${name.replace(/\./g, '_')}`;
+
+  const checkAllowedFileSize = (file: File) => {
+    if (file.size > fileLimitMaxSize) {
+      throw new Error(DEFAULT_ERROR_MESSAGE_MAX_FILE_SIZE);
```

Comment:
There is no option for how can I override the default error messages. There should be an option/prop through which I can pass my own or translated error messages.

Reply:
done